packages/roosterjs-editor-core/lib/undo/UndoSnapshots.ts

```diff
@@ -27,6 +44,10 @@ export default class UndoSnapshots {
         }
     }
 
+    /**
+     * Add a snapshot to the stack
+     * @param snapshot The snapshot to add
+     */
     public addSnapshot(snapshot: string) {
```

Comment:
Discussed with Jiuqing offline, if a function or a method doesn't have a return value, we don't need to add "void". Suggest you to remove all the unused void while you're adding the signatures for all the functions and methods.

Another thing is about the return value. I don't see any comments about the return value of the function or method, please add them as well.

Reply:
Removed all the 'void' in the changed packages. Added comments for all the return value.